projects/orquestra-workflow-runtime/pyproject.toml

```diff
@@ -45,14 +45,13 @@ dependencies = [
 
 [project.optional-dependencies]
 ray = [
-    "ray==2.30.0",
+    "ray[default]==2.30.0",
     "virtualenv>=20.0.24, !=20.21.1",  # Ray depends on this, but it's missing from the base requirements.
     "async-timeout",       # Ray depends on this, but its missing from ray requirements. Can be removed when https://github.com/ray-project/ray/issues/41267 is fixed.
 ]
 
 dashboard = [
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏we no longer need this optional extra. Should we leave it for backwards compat or remove it?

Reply:
Yea we use it in SDK package as well, so for backward compat I left it. Ill add a comment as well why it is here